### PR TITLE
Itsadssd 28081 button uplift

### PR DESCRIPTION
**Pull request by uqtgeor4**
Consolidated buttons in Design system and Future students.
Doc Link below 
https://teams.microsoft.com/l/file/eadcccfc-327e-4b2c-a467-160599c6085f?tenantId=b6e377cf-9db3-46cb-91a2-fad9605bb15c&fileType=docx&objectUrl=https%3A%2F%2Fuq.sharepoint.com%2Fteams%2FLmD6Nzqn%2FShared%20Documents%2FADS%20Design%20team%2FButton%20consolidation%20of%20variants%20in%20FS%20%26%20DS.docx&baseUrl=https%3A%2F%2Fuq.sharepoint.com%2Fteams%2FLmD6Nzqn&serviceName=teams&threadId=19:43865c84b05c4dd58dd852228b29f8d0@thread.skype&messageId=1610666929223&groupId=2d5a5ae4-8744-467d-a20f-88ea63e89e10

### CODE REVIEW

File: packages/storybook-html/src/components/button/button.docs.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    </tr>
+    <tr className="uqds-table__row">
+      <td className="uqds-table__item" data-title="Category:">Secondary light</td>
+      <td className="uqds-table__item" data-title="Styles:">Normal, outline, text, expand</td>
+      <td className="uqds-table__item" data-title="How/When to use:">Used on dark backgrounds</td>
+    </tr>
+    <tr className="uqds-table__row">
+      <td className="uqds-table__item" data-title="Category:">Purple</td>
+      <td className="uqds-table__item" data-title="Styles:">Normal, outline, text, expand</td>
+      <td className="uqds-table__item" data-title="How/When to use:">Used in UQ events and stories sections</td>
```
Comment (uqnm) [inline packages/storybook-html/src/components/button/button.docs.mdx:132]: id be interested to know what the team think of this for how/when to use the purple. did jen or laura provide any input regarding their thinking of when to apply purple?

**Review by uqnm — APPROVED** [review]
Looks good - just interested to see what others thing as per my comment

File: packages/storybook-html/src/components/button/button.docs.mdx (hunk 1 above)
Comment (Bcrunk) [reply packages/storybook-html/src/components/button/button.docs.mdx]: I have run it past Jen again. I think there is another pass required to resolve the use cases and scenarios, I will generate some new tickets to cover additional work.

**Comment by uqmchu** [comment]
This PR should have been rejected due to commit [529f635](https://github.com/uq-its-ss/design-system/pull/151/commits/529f6356319d388a8668b7acdb5c5d0ea0a36b98). Manually fixed the branch and re-merged into master.

**Pull request merged**